gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Jul 02 12:25:17 BST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```

Comment:
by mentioning gradle upgrade i meant as a separate pr ;). surely from 5.6.2 to 6.5.1 there are some extra deprecations, perhaps bin commands have changed and, of cource, wrapper jar file too!